natspec/l2-token/BurnMintERC20.md

```
@@ -0,0 +1,692 @@
+
+
+
+*Uses OZ ERC20 _burn to disallow burning from address(0).Decreases the total supply.*
+
+#### Parameters
+
+| Name | Type | Description |
+|---|---|---|
+| amount | uint256 | undefined |
+
+### burn
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)